tests/config.rs

```diff
@@ -0,0 +1,56 @@
+use assert_cmd::prelude::*;
+use std::env;
+use std::fs;
+use std::io::prelude::*;
+use std::process::{Child, Command, Stdio};
+
+// Does not work on Windows because of the home_dir resolution
```

Comment:
could you check $HOME yourself in the config command (instead of using dirs::home_dir) when `cfg!(test)` is true on windows?, that way you can still configure the home dir on windows to run this test?

Reply:
Maybe CONFIG_LOCATION could be an env variable. It will complicate the logic then in https://github.com/cloudflare/wrangler/pull/221 (for XDG support).

Reply:
Thinking about this, I think it makes sense to copy [cargo's behavior](https://doc.rust-lang.org/cargo/reference/environment-variables.html) and have a $WRANGLER_HOME that defines the location of `.wrangler`... then people can put their stuff wherever they want, and the tests can configure it as needed

Reply:
Good idea